tests/src/Context/EcasContext.php

```diff
@@ -76,26 +85,31 @@ public function ecasUserCreate($user) {
       ))
       ->execute();
     $user->ecas = 1;
-    $this->users[$user->name] = $this->user = $user;
+
     return $user;
   }
 
   /**
    * Remove any created ECAS users.
    *
+   * @see RawDrupalContext::cleanUsers()
+   *
    * @AfterScenario @Ecas
    */
-  public function cleanEcasUsers() {
+  public function ecasCleanUsers() {
     // Remove any users that were created.
-    if (!empty($this->users)) {
-      foreach ($this->users as $user) {
+    if ($this->userManager->hasUsers()) {
```

Comment:
$this->user is not allowed. $this->userManager is still allowed.
But it's better like you're saying, I'll follow your suggestion.

I've also removed one "if", because handling the object with getter $this->getUserManager() we're sure we get always an array.